app.py

```diff
@@ -55,36 +64,42 @@ async def decorated_function(*args, **kwargs):
 
 
 # Get All Users
-@app.get('/users')
+@app.get("/users")
 async def get_users():
     pool = await create_pool()
```

Comment:
IIf you used a class, instead function, you can simplify this query functionalities.